apps/connect/src/hooks/useSyncStatus.ts

```diff
@@ -1,8 +1,12 @@
 import { SharingType } from '@powerhousedao/design-system';
+import { SyncStatus } from 'document-drive';
 import { useSyncExternalStore } from 'react';
 import { useDocumentDriveServer } from './useDocumentDriveServer';
 
-export function useSyncStatus(driveId: string, documentId?: string) {
+export function useSyncStatus(
```

Comment:
in cases where a function's inferred return type is a type imported from elsewhere, we need to import the type and use an explicit type annotation. this is because otherwise when the code is built, the actual type definition for the inferred return type will not be present, meaning that an external user of the code will not be able to have their return type inferred.